w_common/lib/src/common/cache/cache.dart

```diff
@@ -150,22 +150,22 @@ class Cache<TIdentifier, TValue> extends Object with Disposable {
 
   /// The stream of [CacheContext]s that dispatches when an item is updated in
   /// the cache.
-  Stream<CacheContext<TIdentifier, TValue>> get didUpdate =>
+  Stream<CacheContext<TIdentifier, TValue?>> get didUpdate =>
       _didUpdateController.stream;
 
   /// Keys that have not been released.
   Iterable<TIdentifier> get liveKeys =>
-      _cache.keys.where((TIdentifier key) => !_isReleased[key]);
+      _cache.keys.where((TIdentifier key) => !_isReleased[key]!);
 
   /// Values that have not been released.
   ///
   /// To access a released value a [get] or [getAsync] should be used.
-  Future<Iterable<TValue>> get liveValues =>
-      Future.wait(liveKeys.map((TIdentifier key) => _cache[key]));
+  Future<Iterable<TValue>> get liveValues => Future.wait(
+      liveKeys.map((TIdentifier key) => _cache[key]!.then((value) => value!)));
```

Comment:
The type of `_cache` is `Map<TIdentifier, Future<TValue>>`. The `TValue` is non-nullable so we don't need the `.then((value) => value!)`

```suggestion
      liveKeys.map((TIdentifier key) => _cache[key]!));
```